WcaOnRails/db/migrate/20180623171213_add_registration_requirements_columns_to_competitions.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddRegistrationRequirementsColumnsToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :free_competition_enabled, :boolean
+    add_column :Competitions, :on_the_spot_registration_enabled, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee_lowest_denomination, :integer, null: false, default: 0
+    add_column :Competitions, :refund_policy, :boolean
+    add_column :Competitions, :refund_policy_amount, :integer, null: false, default: 0
+    add_column :Competitions, :refund_policy_limit_date, :datetime
+    add_column :Competitions, :guests_entry_fee_enabled, :boolean
+    add_column :Competitions, :guests_entry_fee, :integer, null: false, default: 0
+    add_column :Competitions, :guests_entry_fee_type, :boolean
```

Comment:
This should not be a boolean column. What does it mean for `guests_entry_fee_type` to be true? I think we should make this an enum. You can check out our registration status column for an example of how to do that.

Reply:
I'll check the example, as it probably makes more sense to have it like that.